app/views/foods/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>RESULTS FOR <%= @info.query %></h1>
+Total Results: <%= @info.total_hits %><br><br>
+
+<% @results.first(10).each do |result| %>
```

Comment:
You could limit `@results` to 10 earlier, maybe in the facade, to avoid passing a bunch of data to the view that will never be displayed.